examples/mixed-sources.yaml

```diff
@@ -0,0 +1,23 @@
+- description:
```

Comment:
Good idea to have a catalog like this. Maybe update the names and descriptions a bit?

Reply:
Oops, I forgot to push my change before merging this PR so I've opened https://github.com/canonical/subiquity/pull/1507